src/app/clusters/ota-requestor/ota-requestor.h

```diff
@@ -0,0 +1,42 @@
+/*
+ *
+ *    Copyright (c) 2021 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+/* This file contains the declarations for the Matter OTA Requestor implementation and API.
+ * Applications implementing the OTA Requestor functionality must include this file.
+ */
+
+#include <ota-requestor-driver.h>
+#include <ota-requestor-interface.h>
+
+#pragma once
+
+// This class implements all of the core logic of the OTA Requestor
+class OTARequestor : public OTARequestorInterface
```

Comment:
Why is it necessary to have both `OTARequestor` and `OTARequestorInterface`?

Reply:
Having OTARequestorInterface allows a custom implementation to fully replace the (SDK-supplied) OTARequestor

Reply:
So OTARequestor is our reference/suggested implementation?

Reply:
Yes